src/app/components/demo-components.service.ts

```diff
@@ -123,6 +123,24 @@ export class SkyDemoComponentsService {
         },
         getCodeFiles: () => this.getDemoFiles('Checkbox')
       },
+      {
+        name: 'Code',
```

Comment:
That will require a PR to the [sky-skysp-svc](https://dev.azure.com/blackbaud/Products/_git/sky-skysp-svc) in ADO to show up.

Reply:
OK. Couldn't remember if that was the case for non-phase 2 docs. I'll go ahead and resolve this comment since we'll need to address that after merging this.